web/client/examples/featuregrid/components/FeatureGridActionsSupport.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+const {connect} = require('react-redux');
+const {bindActionCreators} = require('redux');
+const ol = require('openlayers');
+const {isEqual, isUndefined} = require("lodash");
+const {changeMapView} = require('../../../actions/map');
+const mapUtils = require('../../../utils/MapUtils');
+const configUtils = require('../../../utils/ConfigUtils');
+
+const FeatureGridActionsSupport = React.createClass({
+    propTypes: {
+        featuregrid: React.PropTypes.object,
+        actions: React.PropTypes.object,
+        map: React.PropTypes.object
+    },
+    defaultProps: {
+        featuregrid: {},
+        actions: {}
+    },
+    componentDidMount() {
+        this.geoJSON = new ol.format.GeoJSON();
+    },
+    shouldComponentUpdate(nextProps) {
+        return (!isUndefined(nextProps.featuregrid.zoomto) && !isEqual(nextProps.featuregrid.zoomto, this.props.featuregrid.zoomto));
+    },
+    componentDidUpdate() {
+        if (this.props.featuregrid && this.props.featuregrid.zoomto && this.props.featuregrid.zoomto.length > 0) {
```

Comment:
I would move this logic in FeatureGrid, and remove ol explicit dependencies, let's use CoordinateUtils